edb/ir/typeutils.py

```diff
@@ -196,6 +196,28 @@ def is_persistent_tuple(typeref: irast.TypeRef) -> bool:
         return False
 
 
+def get_custom_serialization(
+    typeref: irast.TypeRef,
+) -> Optional[str]:
+    # FIXME: instead of hardcode we need to extract this from the
+    # schema/extension
+    if typeref.real_base_type.id in {
+        s_obj.get_known_type_id('ext::postgis::box2d'),
```

Comment:
Let's use `name_hint` instead of adding known type ids for this.

Reply:
Replaced with `name_hint`